src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -290,7 +291,9 @@ export const useSearch = (services: DiscoverViewServices) => {
         chrome.recentlyAccessed.add(
           savedSearchInstance.getFullPath(),
           savedSearchInstance.title,
-          savedSearchInstance.id
+          savedSearchInstance.id,
+          savedSearchInstance.getOpenSearchType(),
+          moment(Date.now()).format('MM/DD/YYYY HH:mm')
```

Comment:
Can we use timestamp(number) here? It is more convenient to do the sort by using that. And date format should comply with user's localization setting and can be done when render the timestamp.